server/ip-tools.ts

```diff
@@ -166,25 +174,29 @@ export const IPTools = new class {
 		const [ip, bits] = cidrParts;
 		return this.ipRegex.test(ip.trim()) && /^[1-3]?[0-9]$/.test(bits.trim());
 	}
-	/** does not check for validity; use `validRange` for that; supports both range formats */
 	stringToRange(range: string): AddressRange | null {
-		if (!range) return null;
+		if (!IPTools.isValidRange(range)) return null;
```

Comment:
```suggestion
```

I'm pretty sure this is no longer necessary, and also `isValidRange` might be replaceable with `stringToRange(str) !== null`. If you have free time, you might want to benchmark that to see if it's faster.